pkg/infrastructure/services/configmap.go

```diff
@@ -51,24 +54,40 @@ func GetAppPropConfigMapContentHash(name, namespace string, cli *client.Client)
 		return "", nil, err
 	}
 
+	appPropsToApply := map[string]string{}
 	if exist {
-		if _, ok := configMap.Data[appPropContentKey]; !ok {
-			exist = false
+		if _, ok := configMap.Data[appPropContentKey]; ok {
```

Comment:
Can you place this code to separate function to make the function flow easier to read?
Something like getApplicationProperties(configMap &corev1.ConfigMap)